metrics/deploy/lib/cli/flutter/flutter_command.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:process_run/process_run.dart' as cmd;
+
+/// class wrapping up flutter CLI
```

Comment:
Please, format the documentation as sentences (see [effective dart](https://dart.dev/guides/language/effective-dart/documentation#do-format-comments-like-sentences) suggestion). The same for all the rest classes.
```suggestion
/// A wrapper class for the Flutter CLI.
```